assets/stylesheets/bootstrap/_buttons.scss

```diff
@@ -67,21 +67,16 @@ a.btn {
 // --------------------------------------------------
 
 .btn-default,
+.btn-info,
 .btn-success,
 .btn-warning {
   @include button-variant-plain($btn-default-color, $btn-default-bg);
 }
-// Appears in Talis turqoise
-.btn-primary {
-  @include button-variant($btn-primary-color, $btn-primary-bg, $btn-primary-border);
-}
 
-// Info appears as pink
```

Comment:
Broken window: this comment is incorrect, and `btn-info` is actually exactly the same as `btn-default`, `btn-success` and `btn-warning` above